src/overlays/actors/ovl_En_Rz/z_en_rz.c

```diff
@@ -184,16 +191,29 @@ void EnRz_ActorShadowFunc(Actor* thisx, Lights* mapper, PlayState* play) {
 }
 
 void EnRz_ChangeAnim(PlayState* play, EnRz* this, s16 animIndex, u8 animMode, f32 transitionRate) {
+    static AnimationHeader* D_80BFCD20[] = {
+        &object_rz_Anim_00457C, &object_rz_Anim_003A20, &object_rz_Anim_005E50, &object_rz_Anim_003098,
+        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    };
+
+    static AnimationHeader* D_80BFCD3C[] = {
+        &object_rz_Anim_003A20, &object_rz_Anim_003A20, &object_rz_Anim_005390, &object_rz_Anim_003098,
+        &object_rz_Anim_00059C, &object_rz_Anim_000DE8, &object_rz_Anim_0028D4,
+    };
+
+    static LinkAnimationHeader* sLinkAnimations[] = { &gPlayerAnim_link_normal_wait_free,
+                                                      &gPlayerAnim_alink_dance_loop };
     f32 endFrame;
     AnimationHeader** animationPtr;
 
-    if (!EN_RZ_GET_PARAM_8000(&this->actor)) {
+    if (!EN_RZ_GET_SISTER(&this->actor)) {
```

Comment:
This isn't a boolean/flag check any more, compare it to the enum (and comment what the other value is for the else). Applies in several other places.

Reply:
This is still a boolean, are you mistaking it with TYPE ?

Reply:
No, it's now used as an enum with two values, not the same thing.

Reply:
`EN_RZ_GET_SISTER(&this->actor) == ROSA_SISTERS_JUDO`, this one probably is.